staging/src/k8s.io/apiserver/pkg/storage/etcd/etcd_helper.go

```diff
@@ -586,6 +586,10 @@ func (h *etcdHelper) GuaranteedUpdate(
 	}
 }
 
+func (*etcdHelper) Count(pathPerfix string) (int64, error) {
+	return 0, fmt.Errorf("Count is unimplemented for etcd2!")
```

Comment:
is this an error? If we want to distinguish between error and "not implement" we should have at least a special pre-defined error type.

Reply:
Agreed, but I guess we'd need to add it to `apimachinery/pkg/api/errors` to be consistently used. I can add it in separate PR, as I expect a lot of discussions around that.

Reply:
Alternative: convention that -1 means "not implemented"

Reply:
-1 currently means "no data", as per @liggitt request. Also we're not writing in C to do such things;)